### PR TITLE
Remove App switcher

### PR DESCRIPTION
**Pull request by PanSpagetka**
After a talk with @ryelo we decided that it would be better to just hide it in beta.
Screenshot from 2021-01-13 10-49-36

https://issues.redhat.com/browse/RHCLOUD-10692

### CODE REVIEW

**Comment by Hyperkid123** [comment]
Merge only after https://github.com/RedHatInsights/insights-chrome/pull/1101 (contains additional travis config to not publish to QA)

**Comment by ryelo** [comment]
@PanSpagetka We'll want to remove the bundle switcher, which you've done, but we still need to leave the "title" of the bundle where that area was, just remove the dropdown portion of it

**Comment by Hyperkid123** [comment]
@PanSpagetka can you rebase the branch?

**Review by Hyperkid123 — DISMISSED** [review]

File: src/js/App/Sidenav/SideNav.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.ins-c-app-title {
+    color: white;
+    padding: calc(8px + var(--pf-global--spacer--sm)) 12px;
+    width: 100%;
+    margin-top: 7px;
+    border-bottom: 3px solid #3c3f42;
```
Comment (karelhala) [inline src/js/App/Sidenav/SideNav.scss:6]: Can you change this to use PF css vars?
```suggestion
    border-bottom: var(--pf-global--spacer--xs) solid var(--pf-global--Color--300);
```

File: src/js/App/Sidenav/SideNav.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.ins-c-app-title {
+    color: white;
+    padding: calc(8px + var(--pf-global--spacer--sm)) 12px;
+    width: 100%;
+    margin-top: 7px;
```
Comment (karelhala) [inline src/js/App/Sidenav/SideNav.scss:5]:
```suggestion
    margin-top: var(--pf-global--spacer--sm);
```

**Review by karelhala — COMMENTED** [review]
Looking good! Just a few changes to use PF vars instead of direct values.

**Review by ryelo — CHANGES_REQUESTED** [review]
Can you line up the left side of the Bundle name with the rest of the left most content (hamburger + items in the nav)?

**Review by karelhala — APPROVED** [review]

**Review by ryelo — APPROVED** [review]

**Pull request merged**